pom.xml

```diff
@@ -2682,13 +2684,29 @@
       </properties>
     </profile>
     <profile>
-      <id>java9-or-later</id>
+      <id>java11-20</id>
       <activation>
-        <jdk>[9,]</jdk>
+        <jdk>[11,20]</jdk>
       </activation>
       <properties>
+        <aspectj.version>${aspectj.java11.version}</aspectj.version>
         <!-- supported since Java 9 -->
         <maven.compiler.release>${javac.version}</maven.compiler.release>
+        <maven.compiler.source>${javac.version}</maven.compiler.source>
+        <maven.compiler.target>${javac.version}</maven.compiler.target>
```

Comment:
Now these two properties are the same across all JDK version profiles, so could be moved to global `properties`.  The profile `java8` can then be removed.